### DISCUSSION TITLE
Converting PowerPoint to Quarto

### DISCUSSION THREAD

**Discussion by plessl** [quarto-dev/quarto-cli#6224] [Q&A]
### Description

Hi there

I'm planning my workflow to convert comprehensive PowerPoint presentations (PPTX format) to quarto to create lecture notes.

If I haven't missed anything, Quarto doesn't support this feature yet.

Did anyone find a tool that does a reasonably good job?

Thanks,
 Christian

**Comment by mcanouil** [comment]
> If I haven't missed anything, Quarto doesn't support this feature yet.

Quarto is about using markdown/notebook to write content in various formats, not the other way around, so very unlikely it will ever support this feature.

> Did anyone find a tool that does a reasonably good job?

It's called Pandoc (the tool behind Quarto) which does "good job" negatively correlated with the complexity of your slides, see <https://pandoc.org/>.

**Comment by kapsner** [comment]
@plessl I am using the python based [pptx2md](https://github.com/ssine/pptx2md) as an initial step when migrating my PowerPoint slide decks to quarto (there is even a `--qmd` flag providing output in quarto syntax).

However, for me, this is just the first step followed by extensive manual adjustments to have a nicely looking revealjs presentation.
